project_key/models/project_project.py

```
@@ -158,13 +158,14 @@ def generate_project_key(self, text):
         return self._generate_project_unique_key("".join(key))
 
     def _generate_project_unique_key(self, text):
+        self_context = self.with_context(active_test=False)
         res = text
         unique_key = False
         counter = 0
         while not unique_key:
             if counter != 0:
                 res = "%s%s" % (text, counter)
```

Ruff (0.11.9)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)